src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -303,6 +308,26 @@ static MvcCoreLoggerExtensions()
                 LogLevel.Warning,
                 2,
                 "Unable to unambiguously infer binding sources for parameters on '{ActionName}'. More than one parameter may be inferred to bound from body.");
+
+            _unsupportedFormatFilterContentType = LoggerMessage.Define<string>(
+                LogLevel.Trace,
+                1,
+                "Could not find a formatter which supports the content type format '{FormatFilterContentType}'.");
```

Comment:
This isn't really accurate, it should say `cannot find a media type for the format {..}`